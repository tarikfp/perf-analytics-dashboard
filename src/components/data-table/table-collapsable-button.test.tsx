import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TableCollapsableCell from './table-collapsable-button';

const props = {
  isOpen: true,
  onClick: jest.fn(),
};

test('table collapsable cell renders correctly', () => {
  const tree = renderer.create(<TableCollapsableCell {...props} />);
  expect(tree).toMatchSnapshot();
});
