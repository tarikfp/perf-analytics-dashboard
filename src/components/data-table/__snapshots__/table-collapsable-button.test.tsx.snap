// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`table collapsable cell renders correctly 1`] = `
<td
  aria-sort={null}
  className="MuiTableCell-root MuiTableCell-alignLeft MuiTableCell-sizeMedium css-bsph2u-MuiTableCell-root"
>
  <button
    className="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeSmall css-1pe4mpk-MuiButtonBase-root-MuiIconButton-root"
    disabled={false}
    onBlur={[Function]}
    onClick={[MockFunction]}
    onContextMenu={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
      data-testid="KeyboardArrowUpIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6z"
      />
    </svg>
  </button>
</td>
`;
